test/benchmark/README.md

```diff
@@ -0,0 +1,96 @@
+Note: the following benchmarking suite requires an AWS setup, and is an expensive operation involving several high-compute ec2 instances.
+
+This apache-bench based benchmark aims to do a multiple batch-size based inference benchmarking as per provided config in a *.yaml file. Each config represents one model.
+
+Check out a sample vgg11 model config at the path: `tests/suite/vgg11.yaml`
+
+### Setup
+
+* Ensure you have access to an AWS account i.e. [setup](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html) your environment such that awscli can access your account via either an IAM user or an IAM role. An IAM role is recommended for use with AWS. For the purposes of testing in your personal account, the following managed permissions should suffice: <br>
+-- [AmazonEC2ContainerRegistryFullAccess](https://console.aws.amazon.com/iam/home#policies/arn:aws:iam::aws:policy/AmazonEC2ContainerRegistryFullAccess) <br>
+-- [AmazonEC2FullAccess](https://console.aws.amazon.com/iam/home#policies/arn:aws:iam::aws:policy/AmazonEC2FullAccess) <br>
+-- [AmazonS3FullAccess](https://console.aws.amazon.com/iam/home#policies/arn:aws:iam::aws:policy/AmazonS3FullAccess) <br>
+* [Create](https://docs.aws.amazon.com/cli/latest/reference/ecr/create-repository.html) an ECR repository with the name “torchserve-benchmark” in the us-west-2 region
+* Ensure you have [docker](https://docs.docker.com/get-docker/) client set-up on your system - osx/ec2
+* Adjust the following global variables to your preference in the file serve/test/benchmark/tests/utils/__init__.py <br>
+-- IAM_INSTANCE_PROFILE :this role is attached to all ec2 instances created as part of the benchmarking process. Create this as described [here](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html#create-iam-role). Default role name is 'EC2Admin'.<br>
+-- S3_BUCKET_BENCHMARK_ARTIFACTS :all temporary benchmarking artifacts including server logs will be stored in this bucket: <br>
+-- DEFAULT_DOCKER_DEV_ECR_REPO :docker image used for benchmarking will be pushed to this repo <br>
+
+*The following steps assume that the current working directory is serve/.*
+
+1. Create or use any python virtual environment
+```
+python3 -m venv bvenv
+source bvenv/bin/activate
+```
+2. Install requirements for the benchmarking 
+```
+pip install -r test/benchmark/requirements.txt
+```
+3. Make sure that you've setup AWS account correctly
+```
+aws sts get-caller-identity
```

Comment:
When setting up base instance make sure to set IAM role to EC2ADmin with the permissions above

Maybe add
* SSH port and with all inbound access

Reply:
Thanks, it wouldn't be necessary for every user to use EC2Admin as an ec2-attached-iam-role, user can chose their own method for the first setup. So I'd leave that part out.

Updated the readme regarding ssh port.